ini/INIReader.h

```diff
@@ -263,13 +266,22 @@ class INIReader
 
 protected:
     int _error;
-    std::map<std::string, std::string> _values;
-    std::set<std::string> _sections;
-    static std::string MakeKey(std::string section, std::string name);
+    std::map<std::string, std::map<std::string, std::string>> _values;

```

Comment:
use `unordered_map` which is faster, ref: https://www.geeksforgeeks.org/unordered_map-in-cpp-stl/
> Internally unordered_map is implemented using Hash Table, the key provided to map are hashed into indices of hash table that is why performance of data structure depends on hash function a lot but on an average the cost of search, insert and delete from hash table is O(1).

Reply:
or `unorder_set`